assets/js/googlesitekit-idea-hub-post-list-notice.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Idea Hub Post List notice.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { CORE_USER } from './googlesitekit/datastore/user/constants';
+const { dispatch } = Data;
+
+const WEEK_IN_SECONDS = 3600 * 24 * 7;
+
+domReady( () => {
+	const newNotice = document.getElementById( 'googlesitekit-notice-new' );
+	const savedNotice = document.getElementById( 'googlesitekit-notice-saved' );
```

Comment:
Per my last comment about notice slugs, these IDs would need to be updated accordingly.

Also, I think we could simplify the logic here by using a combined selector:

```js
document.querySelector( '[id="googlesitekit-notice-A"],[id="googlesitekit-notice-B"]' )`
```

Then the slug can be parsed from the `el.id` that comes back by stripping off the leading `googlesitekit-notice-`, although I wouldn't be against adding the notice slug as a data attribute to `Notice`.

Reply:
https://github.com/google/site-kit-wp/pull/3668/commits/e4a3fea58fb52114f5aef655f1ce6dfb1eeef9a0